src/script/lua_api/l_object.h

```diff
@@ -173,6 +173,12 @@ class ObjectRef : public ModApiBase {
 	// get_acceleration(self)
 	static int l_get_acceleration(lua_State *L);
 
+	// set_rotation(self, pos)
+	static int l_set_rotation(lua_State *L);
+	
+	// get_rotation(self, pos)
```

Comment:
`pos` needs replacement here too (same for new line 176)

Reply:
Done.